website/sidebars.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
       'getting-started',
       'installation',
       'installation-8-20',
+      'project-templates',
```

Comment:
Let's put this after all the tutorials (at the bottom of the Getting started section).